src/Illuminate/Foundation/helpers.php

```diff
@@ -1047,3 +1047,23 @@ function view($view = null, $data = [], $mergeData = [])
         return $factory->make($view, $data, $mergeData);
     }
 }
+
+if (! function_exists('when')) {
+    /**
+     * Output a value if the given condition is true.
+     *
+     * @param  bool  $condition
+     * @param  string  $output,
+     * @param  bool  $escape
+     * @return void
+     *
+     * @throws \Symfony\Component\HttpKernel\Exception\HttpException
+     * @throws \Symfony\Component\HttpKernel\Exception\NotFoundHttpException
+     */
+    function when($condition, $output, $escape = false)
+    {
+        if ($condition) {
+            echo $escape ? e($output) : $output;
```

Comment:
I would make this a return instead of an echo. That way this function can be used in other places, not just Blade templates.

```suggestion
            return $escape ? e($output) : $output;
```